config/locales/administrate.es.yml

```diff
@@ -6,6 +6,8 @@ es:
       destroy: Destruir
       edit: Editar
       show: Mostrar
+      new: Novaj
+      back: Reen
```

Comment:
I've cloned Ben Morgan's fork, added a fix and pushed it to https://github.com/thoughtbot/administrate/compare/translations

Things we can do

* Fork Ben's fork and submit a PR to his branch.
* You can merge this PR and cherry-pick [my last commit](https://github.com/thoughtbot/administrate/commit/680caeeb1d83fc105c59c7f797c4fcbc10dea724) on our repo.
* Ben can cherry-pick my commit from our repo?

```
git fetch thoughtbot-origin
git cherry-pick 680caeeb1d83fc105c59c7f797c4fcbc10dea724
git push
```